protos/escrow.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
```

Comment:
Can you remove this commit from junlin's PR thx

Reply:
removed